### ISSUE TITLE
Consider centrally defining all users of the blob URL store

### ISSUE THREAD

**Issue by annevk** [w3c/FileAPI#93]
Letting the blob URL store be extended doesn't seem like a good idea. At least, I think there's general agreement it's not the best pattern and we don't want to see it spread further.

We should enumerate all objects which can be stored in it and what policies affect them, rather than allowing arbitrary extensions.

Raising this separately from #92 as that's a big enough task on its own.

**Comment by mkruisselbrink** [comment]
Yeah, that might make sense. Not entirely sure what the layering would look like, but we definitely don't want to encourage other specs to extend blob URLs further.

**Comment by annevk** [comment]
This is fixed, right? By @saschanaz in #115. We only allow Blob or MediaSource objects now.
